Detections/MultipleDataSources/NOBELIUM_IOCsMay2021.yaml

```diff
@@ -4,7 +4,17 @@ description: |
   'Identifies a match across various data feeds for domains, hashes and IP IOCs related to NOBELIUM.
   Ref: https://www.microsoft.com/security/blog/2021/05/27/new-sophisticated-email-based-attack-from-nobelium/'
 severity: Medium 
-requiredDataConnectors: 
+requiredDataConnectors:
+  - connectorId: AWSS3
```

Comment:
I think that in the "mutliplesources" rules, when you added network, you missed on DeviceNetworkEvents. Not just here.

Reply:
No. it is not missing even in this file :) see line 44